Sources/Confidence/Confidence.swift

```diff
@@ -81,13 +81,13 @@ public class Confidence: ConfidenceEventSender {
 
     /**
     Fetches latest flag evaluations and store them on disk. Regardless of the fetch outcome (success or failure), this
-    function activates the cache after the fetch.
+    function activates the cache after the fetch. Timeout for this function is configurable.
     Activating the cache means that the flag data on disk is loaded into memory, so consumers can access flag values.
     Fetching is best-effort, so no error is propagated. Errors can still be thrown if something goes wrong access data on disk.
     */
-    public func fetchAndActivate() async throws {
+    public func fetchAndActivate(fetchTimeoutSec: Int = 10) async throws {
```

Comment:
can we have this in milliseconds? I believe it's more common to have duration in mills.